src/utils/server-requests.js

```diff
@@ -58,15 +59,21 @@ export const loginAsGuest = async (
 
     if (res.status == 200 || res.status == 201) {
       dispatch({ type: "LOGIN", payload: res.data });
-    }
 
-    localStorage.setItem("data", JSON.stringify(res.data.foundUser));
-    localStorage.setItem("token", res.data.encodedToken);
-    localStorage.setItem("userName", res.data.foundUser.firstName);
+      localStorage.setItem("data", JSON.stringify(res.data.foundUser));
```

Comment:
use constant instead of using string. it will lead you to typo.

Reply:
Noted! I will change them later on.